.github/README.md

```diff
@@ -0,0 +1,32 @@
+## GitHub Actions
+
+While publishing and testing of the library is handled by [JetBrains TeamCity](https://www.jetbrains.com/teamcity/),
+there are some CI operations that are handled by GitHub actions instead.
+
+### Publishing Docs
+
+The building of the documentation website in [docs](../docs), and the publishing of it along
```

Comment:
Let's add an info that publishing fully replaces previous documentation (we have no versioning)